quickemu

```diff
@@ -0,0 +1,47 @@
+---
+tags: [ virtualization ]
+---
+# To run a vm
+quickemu (--vm <path-to-vm.conf>)
+
+# To run a vm with a spice display
+quickemu (--vm <path-to-vm.conf>) --display spice
+
+# To run a vm with a braille terminal display - Version > 3.11 and working brltty required
+quickemu (--vm <path-to-vm.conf> --accessible braille --display sdl)
+  #   or
```

Comment:
```suggestion
#   or
```

Optional change: indentation level can be decreased.

```suggestion
#   or:
```

The same issue as above.

```suggestion
# or:
```

Optional change: fixes can be applied together.